plugins/woocommerce/tests/legacy/unit-tests/admin/class-wc-tests-admin-dashboard.php

```
@@ -37,6 +37,63 @@ public function tearDown(): void {
+		$this->expectOutputRegex( '/wc-recent-reviews-widget-loading/' );
+		$this->expectOutputRegex( '/wc-recent-reviews-widget-content/' );
+	}
+
+	/**
+	 * Test: recent reviews widget placeholder.
+	 */
+	public function test_recent_reviews_widget_content_new_version() {
+		$product     = WC_Helper_Product::create_simple_product();
+		$product_id  = $product->get_id();
+		$comment_id1 = WC_Helper_Product::create_product_review( $product_id );
+		$comment_id2 = WC_Helper_Product::create_product_review( $product_id );
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$comment_id2'. (undefined)

(UnusedLocalVariable)